src/core/main.c

```diff
@@ -1679,7 +1679,7 @@ int main(int argc, char *argv[]) {
         log_open();
 
         if (arg_show_status == _SHOW_STATUS_UNSET)
-                arg_show_status = SHOW_STATUS_YES;
+                arg_show_status = arg_confirm_spawn ? SHOW_STATUS_NO : SHOW_STATUS_YES;
```

Comment:
well, hmm, i think a better way would be to change the ask_string() call to reprint the question every 500ms or so automatically. This would then solve the problem a lot more comprehensively, as it would apply to all questions we ask, and would result in a refresh of the question regardless which component dumps on the screen. 

Specifically I think ask_string() should prefix its output with `\r` to bring the cursor to the beginning of the terminal line, and then print the message, redoing it every 500ms. As long as nothing interfereres with out output this logic will have no visible effect as we constantly overprint the visible text with the exact same text. However, if something is dumped in the middle, then our question won't get lost, as we'll ask soon again...